### PR TITLE
fix(ui5-bar): correct padding values in dialog

### PR DESCRIPTION
**Pull request by hinzzx**
Based on the design specifications, the intended padding for both the left and right sides of the `ui5-bar` should be 16px. However, upon observation, it is evident that the actual padding measures only 12px ( 0.5rem padding of the endcontent container + 0.25rem margin of the slotted element ).

With this changes, the paddings are now correct.

Fixes: #6908

### CODE REVIEW

File: packages/fiori/src/themes/Bar.css [hunk 1]
```diff
@@ -18,15 +18,15 @@
...
+	padding-inline-start: 0.75rem;
 	display: flex;
 	flex-direction: row;
 	align-items: center;
 	justify-content: flex-start;
 }
 
 .ui5-bar-root .ui5-bar-content-container {
-	min-width: 30%;
+	min-width: 29%; /* Remaining space after removing paddings and margins */
```
Comment (ilhan007) [inline packages/fiori/src/themes/Bar.css:29]: 29% looks magic, and could lead to inaccuracies. Isn't better to use the percent that you know is allocated for this area - let's say 30 and remove the total of star end paddings (if they are known), for example calc(30% - 1.5rem)

File: packages/fiori/src/themes/Bar.css [hunk 2]
```diff
@@ -18,15 +18,15 @@
...
+	padding-inline-start: 0.75rem;
 	display: flex;
 	flex-direction: row;
 	align-items: center;
 	justify-content: flex-start;
 }
 
 .ui5-bar-root .ui5-bar-content-container {
-	min-width: 30%;
+	min-width: calc(30% - 4rem); /* Remaining space after removing paddings and margins */
```
Comment (nnaydenow) [inline packages/fiori/src/themes/Bar.css:29]: Why 4rem? How did we calculate this? Can we replace this 4rem with a sum of 4-5 variables for each part of the calculation?

Comment (hinzzx) [reply packages/fiori/src/themes/Bar.css:29]: The way I calculated the 4 rem was:
**2 x 0.75rem - // 1.5rem**
```
.ui5-bar-root .ui5-bar-startcontent-container {
	padding-inline-start: 0.75rem;
	
.ui5-bar-root .ui5-bar-endcontent-container {
	padding-inline-end: 0.75rem;
```

Following up -
**1.5rem + 1rem which comes from** - 

```
.ui5-bar-root .ui5-bar-midcontent-container {
	padding-left: 0.5rem;
	padding-right: 0.5rem;
```

**Current total sum - 2.5rem**

And now I finally add 1.5rem from here: // Which looking at it now is I think the case for the wrong calculation. Am I correct or the error is somewhere else ?

```
::slotted(*) {
	margin: 0 0.25rem; // for each slotted element in the start,mid or end we have 0.5rem - 0.25rem for each left and right
}
```
 **0.5rem x 3 = 1.5rem => 2.5rem + 1.5rem = 4rem;**

Comment (nnaydenow) [reply packages/fiori/src/themes/Bar.css:29]: Extract these numbers in variables and use calc() function.

**Review by tsanislavgatev — APPROVED** [review]

**Pull request merged**